secretmanager/src/create_secret_with_user_managed_replication.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/*
+ * Copyright 2022 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * For instructions on how to run the full sample:
+ *
+ * @see https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/secretmanager/README.md
+ */
+
+declare(strict_types=1);
+
+namespace Google\Cloud\Samples\SecretManager;
+
+// Import the Secret Manager client library.
+use Google\Cloud\SecretManager\V1\Replication;
+use Google\Cloud\SecretManager\V1\Replication\UserManaged;
+use Google\Cloud\SecretManager\V1\Replication\UserManaged\Replica;
+use Google\Cloud\SecretManager\V1\Secret;
+use Google\Cloud\SecretManager\V1\SecretManagerServiceClient;
+
+/**
+ * @param string $projectId Your Google Cloud Project ID (e.g. 'my-project')
+ * @param string $secretId  Your secret ID (e.g. 'my-secret')
+ * @param array  $locations Replication locations (e.g. array('us-east1', 'us-east4'))
+ */
+function create_secret_with_user_managed_replication(string $projectId, string $secretId, array $locations): void
+{
+    // Create the Secret Manager client.
+    $client = new SecretManagerServiceClient();
+
+    // Build the resource name of the parent project.
+    $parent = $client->projectName($projectId);
+
+    $replicas = array();
+    foreach ($locations as $location) {
+        $replica = new Replica();
+        $replica->setLocation($location);
+        array_push($replicas, $replica);
+    }
+
+    $secret = new Secret([
+        'replication' => new Replication([
+            'user_managed' => new UserManaged(),
+        ]),
+    ]);
+
+    $secret->getReplication()->getUserManaged()->setReplicas($replicas);
```

Comment:
Why not be consistent here?
```suggestion
    $secret = new Secret([
        'replication' => new Replication([
            'user_managed' => new UserManaged([
                'replicas' => $replicas
            ]),
        ]),
    ]);
```